### ISSUE TITLE
Chrome-launcher typescript compiler, can't find name

### ISSUE THREAD

**Issue by pedro93** [GoogleChrome/lighthouse#2497]
After adding chrome-launcher as a module to the [pwmetrics](https://github.com/paulirish/pwmetrics) project I came across this typescript issue:
```
> pwmetrics@2.0.1 build /home/pedro.cardoso/dev/pwmetrics
> tsc

lib/index.ts(36,13): error TS2304: Cannot find name 'chromeLauncher'.
lib/index.ts(146,33): error TS2503: Cannot find namespace 'chromeLauncher'.
```

The respective code is: 
```js
const chromeLauncher = require('chrome-launcher');
....

 async launchChrome(): Promise<chromeLauncher.LaunchedChrome|Error> {
    try {
      console.log(messages.getMessage('LAUNCHING_CHROME'));
      this.launcher = await chromeLauncher.launch({
        port: this.flags.port,
        chromeFlags: this.flags.chromeFlags
      });
      this.flags.port = this.launcher.port;
    } catch(error) {
      await this.killLauncher();
      return error;
    }
  }
```

If I add the project as a js module (not in the docs I know) I get the following:
```
lib/index.ts(5,38): error TS7016: Could not find a declaration file for module 'chrome-launcher'. '/home/pedro.cardoso/dev/pwmetrics/node_modules/chrome-launcher/chrome-launcher.js' implicitly has an 'any' type.
```

I import the module as: 
```js
import {chromeLauncher, LaunchedChrome} from 'chrome-launcher';
....

async launchChrome(): Promise<LaunchedChrome|Error> {
    try {
      console.log(messages.getMessage('LAUNCHING_CHROME'));
      this.launcher = await chromeLauncher.launch({
        port: this.flags.port,
        chromeFlags: this.flags.chromeFlags
      });
      this.flags.port = this.launcher.port;
    } catch(error) {
      await this.killLauncher();
      return error;
    }
  }
```

__System details__: 
Ubuntu 16.04 LTS
Node: V8.0.0
NPM: V5.0.0

__How to reproduce__:
Trying to merge LH.v2 to pwmetrics and add the chrome-launcher dependency :)

Thanks for the time.

**Comment by samccone** [comment]
`chromeLauncher` is not exported from 
https://github.com/GoogleChrome/lighthouse/blob/master/chrome-launcher/chrome-launcher.ts

for typescript consumption do the following


```ts
import {launch, LaunchedChrome} from 'chrome-launcher';

let launched: LaunchedChrome;
launched = launch()'
```

**Comment by pedro93** [comment]
Thanks for looking into this @samccone, i've tried your suggestion but I still get the message:
 
```
lib/index.ts(5,22): error TS7016: Could not find a declaration file for module 'chrome-launcher'. '/home/pedro.cardoso/dev/pwmetrics/node_modules/chrome-launcher/chrome-launcher.js' implicitly has an 'any' type.
```

Here is the travis build: https://travis-ci.org/paulirish/pwmetrics/builds/242740991
and the relevant source code (its incapsulated in index.ts): https://github.com/paulirish/pwmetrics/pull/120/files#diff-13b5b151431c7e7a17f273559ed212d5

I noticed that chrome-launcher is exported as a standard javascript project, and that it doesn't have a explicit types files, could either of these be the cause of the problems? Or can I not use the chrome-launcher types in pwmetrics?

**Comment by denar90** [comment]
`import {launch, LaunchedChrome} from 'chrome-launcher/chrome-launcher';` - solving the problem

**Comment by pedro93** [comment]
Would renaming chrome-launcher.ts to index.ts and changing the package.json[main] property accordingly make this work? The nested import seems counter intuitive.

**Comment by samccone** [comment]
I will investigate how we can better handle this.

**Comment by pedro93** [comment]
May I suggest trying to rename the chrome-launcher files to index and changing accordingly in the package.json?

**Comment by samccone** [comment]
The primary consumption point of this module is going to be javascript so we can not change that bit, however renaming to index.ts might end up fixing the problem as well. ~investigation needed~

**Comment by samccone** [comment]
Going to close this issue as working as intended. we can follow up with investigation around easier typescript requires.

**Comment by paulirish** [comment]
the #2513 fix is shipped in chrome-launcher@0.1.3
